Dockerfile

```diff
@@ -10,23 +10,23 @@ ENV ZABBIX_SERVER "zabbix"
 ENV ZABBIX_HOST "k8s"
 ENV CRYPTOGRAPHY_DONT_BUILD_RUST "1"
 
-COPY --chown=nobody:users requirements.txt /app/requirements.txt
+WORKDIR /app
+COPY --chown=nobody:users Pipfile  /app/
 
-RUN  apt-get update -y && \
-       apt-get install libffi-dev libffi7 libssl-dev bash screen ncdu -y && \
-       pip3 install --upgrade pip && \
-       pip3 install -r /app/requirements.txt && \
-       apt-get upgrade -y && \
-       apt-get dist-upgrade -y  && \
-       apt-get remove base libssl-dev libffi-dev gcc -y && \
-       apt-get autoremove -y && \
-       rm -rf /var/lib/apt/lists/* /root/.cache
+RUN  apt-get update -y
```

Comment:
The idea of the original implementation was to save container layers using a single step. Do you have strong reasons to use dedicated layers?

Reply:
yes, more layers prevents pushing big layers for small changes and to reuse these layers if there are only changes in other layers